Source/PeleLMeX_Data.cpp

```diff
@@ -52,6 +52,7 @@ PeleLM::LevelData::LevelData(
   }
   if (a_nAux > 0) {
     auxiliaries.define(ba, dm, a_nAux, a_nGrowState, MFInfo(), factory);
+    diff_aux_cc.define(ba, dm, a_nAux, a_nGrowState, MFInfo(), factory);
```

Comment:
Maybe it's a number of ghost cell issue? Shouldn't `diff_aux_cc` have the same number of ghost cells as `diff_cc`?

Reply:
I've fixed this, but there were too many ghost cells rather than too few, so I don't think this was an issue.